### PR TITLE
Avoid including unistd.h and replace write(STDERR) with fputs(stderr)

### PR DESCRIPTION
**Pull request by kubamracek**

### CODE REVIEW

**Comment by kubamracek** [comment]
@swift-ci please test

**Comment by kubamracek** [comment]
@swift-ci please test Windows platform

**Review by compnerd — CHANGES_REQUESTED** [review]
The removal of unistd is fine, but this is subtly different.  `write` is just a wrapper over the syscall, `fputs` may buffer.  This needs a `fflush` at least.

**Comment by kubamracek** [comment]
@swift-ci please test

**Comment by kubamracek** [comment]
Added fflush calls.

**Review by compnerd — APPROVED** [review]

**Review by mikeash — APPROVED** [review]

**Comment by kubamracek** [comment]
@swift-ci please test

**Comment by kubamracek** [comment]
@swift-ci please test Linux platform

**Comment by kubamracek** [comment]
@swift-ci please test Windows platform

File: stdlib/public/runtime/Errors.cpp [hunk 1]
```diff
@@ -312,7 +310,8 @@ reportNow(uint32_t flags, const char *message)
 #define STDERR_FILENO 2
   _write(STDERR_FILENO, message, strlen(message));
 #else
-  write(STDERR_FILENO, message, strlen(message));
+  fputs(message, stderr);
+  fflush(stderr);
```
Comment (al45tair) [inline stdlib/public/runtime/Errors.cpp:314]: FWIW, `stderr` is normally unbuffered by default, so the `fflush()` is probably not necessary. It won't hurt though.

**Pull request merged**